src/styles/globals.css

```diff
@@ -1,3 +1,80 @@
+:root {
```

Comment:
Like I have commented [here](https://github.com/AritDeveloperCircle/findyourpeople/pull/60/files#r1181148927), this extensive styling will probably go away once we implement the design chosen from the designers.

Reply:
alright ma